index.css

```diff
@@ -93,3 +96,210 @@ nav .drop-down {
 }
+
+.grid-container {
+  display: grid;
+  grid-template-columns: 100%;
+  row-gap: 60px;
+  padding-inline: 30px;
+  margin-block-start: 70px;
+  margin-block-end: 50px;
+}
+
+.project-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  background-color: white;
+  border-radius: 15px;
+  border: 1px solid rgb(199, 194, 194);
+  padding: 15px;
+}
+
+.project-image img {
+  width: 100%;
+  height: 220px;
+  object-fit: cover;
+  border-radius: 10px;
+}
+
+.project-details {
+  display: block;
+  padding-inline: 7px;
+  margin-block: 12px;
+}
+
+.project-details h3 {
+  font-weight: 700;
+  font-size: 32px;
+  color: #172b4d;
+}
+
+.canopy-para {
+  padding-block: 20px;
+}
+
+.canopy-para span {
+  font-weight: 700;
+  font-size: 13px;
+  color: #344563;
+}
+
+.canopy-para img {
+  padding-inline: 10px;
+}
+
+.canopy-para p {
+  font-weight: 600;
+  font-size: 13px;
+  color: #7a869a;
+}
+
+.mini-description {
+  font-weight: 400;
+  font-size: 15px;
+  color: #344563;
+}
+
+.language-tags {
+  margin-block: 12px;
+}
+
+.language-tags ul {
+  display: flex;
+  justify-content: flex-start;
+  gap: 15px;
+  list-style: none;
+}
+
+.language-tags ul div {
+  padding: 5px;
+  padding-inline: 10px;
+  border-radius: 7px;
+  background-color: #ebebff;
+}
+
+.project-detail-btn button {
+  padding: 12px;
+  border: 1px solid #396df2;
+  font-weight: 500;
+  font-size: 17px;
+  color: #396df2;
+  background-color: white;
+  border-radius: 8px;
+  margin-block: 5px;
+}
+
+.project-detail-btn button:hover {
+  background-color: #6f95f7;
+  color: white;
+}
+
+.project-detail-btn button:active {
+  background-color: #1755f3;
+}
+
+.about-myself-section {
+  min-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  border-radius: 0 80px 0 0;
+  background-color: #fff;
+  padding-block: 114px;
+}
+
+.about-myself-description {
+  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  color: #344563;
+  padding-inline: 30px;
+  margin-top: 30px;
+}
+
+.about-myself-description h2 {
+  font-weight: 700;
+  font-size: 48px;
+  color: #091e42;
+}
+
+.about-me-icons ul {
+  margin-block: 20px;
+}
+
+.about-myself-skills {
+  margin-block: 40px;
+  padding-inline: 30px;
+}
+
+.about-myself-skills ul {
+  list-style: none;
+}
+
+.about-myself-skills ul li {
+  padding-block: 15px;
+  font-weight: 500;
+  font-size: 20px;
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.language-tags ul div li {
+  font-weight: 500;
+  font-size: 12px;
+  color: #6070ff;
+}
+
+.about-myself-skills ul li:nth-of-type(1) i {
+  margin-left: 160px;
+  margin-block: 0;
+  padding-block: 0;
+}
+
+.about-myself-skills ul li:nth-of-type(2) i {
+  margin-left: 156px;
+  margin-block: 0;
+  padding-block: 0;
+}
+
+.about-myself-skills ul li:nth-of-type(3) i {
+  margin-left: 218px;
+  margin-block: 0;
+  padding-block: 0;
+}
```

Comment:
- [OPTIONAL] - I think it's better to use Flex to align the content of skills and icons. Check for use of Justify content with space between.